docker-cloud.yml

```diff
@@ -288,3 +288,15 @@ trial-remover:
     LOGGING_URL:
   tags:
     - opentrials
+
+remove-unknown-documentcloud-docs:
+  image: okibot/processors
```

Comment:
I totally overlooked the fact that we have a bot building docker images from this repo. Neat!